### PR TITLE
Add calculator UI

### PR DESCRIPTION
**Pull request by alonsofl**
### Summary
Develop the calculator UI for the project.

#### Details
- Removed boilerplate from CRA installation.
- Created a React class based component in `Calculator.js`.
- Developed the UI of the calculator.

> Note:
> Using JavaScript linters as recommended in the [PR #1](https://github.com/alonsofl/math-magicians/pull/1).
> GitHub is having issues with [linters checking (actions)](https://www.githubstatus.com/), but linters for the projects pass successfully locally.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```
Comment (mricanho) [inline src/App.js:3]: - [ ] This function component should be class-based. In the same way that you created the calculator class

**Review by mricanho — COMMENTED** [review]
Hi @alonsofl, 

Good job on this milestone! Your gitflow, linters, README file and style are awesome! 🔥🔥

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- [ ] You need to remove all the boilerplate created by the CRA (texts, images, and styles) e.g

imagen


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (alonsofl) [reply src/App.js]: Hello @mricanho,
Thanks for your review. The changes required have been fixed.

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Comment by alonsofl** [comment]
Thanks @arslanbisharat!

**Pull request merged**